catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -101,6 +114,10 @@ public class HadoopCatalogOperations extends ManagedSchemaOperations
 
   private FileSystemProvider defaultFileSystemProvider;
 
+  static {
```

Comment:
You can replace with this static code block with 

xxxx = new ThreadPoolExecutor() {
   {
       allowCoreThreadTimeOut(true);
   }
}